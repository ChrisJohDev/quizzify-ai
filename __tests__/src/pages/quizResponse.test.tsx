import {render, screen }from '@testing-library/react';
import QueryResponse from '@/pages/quizResponse';
import '@testing-library/jest-dom'

describe('QueryResponse - snapshot test', () => {
  it('renders correctly', () => {
    const { container } = render(<QueryResponse queryResponse={{
      questions: [],
      answers: ''
    }} subject={''} />);
    expect(container).toMatchSnapshot();
  });
});