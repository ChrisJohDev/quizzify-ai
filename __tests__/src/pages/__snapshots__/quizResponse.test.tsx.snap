// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QueryResponse - snapshot test renders correctly 1`] = `
<div>
  <main
    class="wrapper"
  >
    <section>
      <h1>
        Query Response
      </h1>
      <p>
        Your quiz on the subject of 
        <strong />
         is ready:
      </p>
    </section>
    <section>
      <div
        class="qAndA qAndA"
      >
        <div
          class="questions"
        >
          <h2>
            Questions
          </h2>
        </div>
        <div
          class="answers"
        >
          <h2>
            Answers
          </h2>
          <pre />
        </div>
      </div>
    </section>
  </main>
</div>
`;
